changelog/cNPiQAqZTEeqDmtRcz-kKQ.md

```diff
@@ -0,0 +1,4 @@
+audience: general
+level: patch
+---
+Generic Worker: Adds validation that the task user is able to read and execute the generic-worker binary. If the task user is not able to read and execute the binary, the task will fail as `exception/internal-error`.
```

Comment:
Let's make this a start up check, so that it does not claim a task if it cannot upload artifacts.